internal/sink/sinkSender.go

```diff
@@ -471,3 +467,25 @@ func (s *sender) CreateMetrics(m metrics.Metrics) {
 	s.maxWorkersGauge = m.ConsumerMaxDeliveryWorkersGauge.With(prometheus.Labels{metrics.UrlLabel: s.id})
 
 }
+
+func getPartnerIds(l ancla.Register) ([]string, error) {
+	switch v := l.(type) {
+	case *ancla.RegistryV1:
+		return v.PartnerIDs, nil
+	case *ancla.RegistryV2:
+		return v.PartnerIds, nil
```

Comment:
maybe we should match the names (partnerids) ... it'll make things simpler in some places

Reply:
so let's change ancla's

```go

type RegistryV2 struct {
	PartnerIds   []string
	Registration webhook.RegistrationV2
}
```

to

```go

type RegistryV2 struct {
	PartnerIDs   []string
	Registration webhook.RegistrationV2
}
```